compiler/src/iree/compiler/Codegen/LLVMGPU/test/transform_dialect_bufferize.mlir

```diff
@@ -2,7 +2,7 @@
 
 hal.executable private @pad_matmul_static_dispatch_0  {
   builtin.module {
-    func.func @pad_matmul_static_dispatch_0(%arg0: tensor<250x500xf32>, %arg1: tensor<500x1020xf32>) -> tensor<250x1020xf32> {
```

Comment:
afaik, dispatch region does not return anything. So it makes sense to me removing that.